client/src/components/ReviewInstance.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import styled from 'styled-components';
+import StarRenderer from './StarRenderer.jsx';
+
+class ReviewInstance extends React.Component {
```

Comment:
Not an issue, but you can use functional components instead of class if you're not using state!